### PR TITLE
ISSD-945 Marketplace] Add app tiered pricing

### PR DESCRIPTION
**Pull request by eliasantosf**
Task: [ISSD-945](https://liferay.atlassian.net/browse/ISSD-945)

### CODE REVIEW

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/InformLicensingTermsPage/InformLicensingTermsPricePage.tsx [hunk 1]
```diff
@@ -21,37 +37,167 @@ export function InformLicensingTermsPricePage({
...
-						const skuBody = {
-							...skuJSON,
-							price: parseFloat(appLicensePrice),
-						};
+						for (const sku of skusJSON?.items) {
+							const skuBody = {
+								...sku,
+								price: getSkuPrice(appLicensePrice, sku),
+							};
+							handlePostPriceEntryIdTierPrice(sku);
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/InformLicensingTermsPage/InformLicensingTermsPricePage.tsx:197]:
```suggestion
							await handlePostPriceEntryIdTierPrice(sku);
```

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

Good job @eliasantosf

**Comment by kevenleone** [comment]
Just starting reviewing :)

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/InformLicensingTermsPage/InformLicensingTermsPricePage.tsx (hunk 1 above)
Comment (eliasantosf) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/InformLicensingTermsPage/InformLicensingTermsPricePage.tsx]: Done!

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:closed

**Pull request closed without merging**